db/seeds.rb

```diff
@@ -5,3 +5,17 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+User.create!(email: "user@example.com", password: "password",
+  password_confirmation: "password")
+
+User.create!(email: "lunk@example.com", password: "password",
+  password_confirmation: "password")
+
+User.create!(email: "zorldo@example.com", password: "password",
+  password_confirmation: "password")
+
```

Comment:
Nice work with the `create!` instead of `create in your seed file. The create method can actually take an array, so sometimes it's more readable to do this kind of thing like this:
```
users_attributes = [
  {email: "user@example.com", password: "password", password_confirmation: "password"},
  {email: "lunk@example.com", password: "password", password_confirmation: "password"},
  {email: "zorldo@example.com", password: "password", password_confirmation: "password"},
]
User.create!(users_attrs)
```
(I didn't check this so apologies if the syntax is a bit off.)